search-app/docs/api.md

```diff
@@ -3047,18 +3046,86 @@ Variable have the following fields:
 
 * variable_name (REQUIRED): free text specifying the key of the variable. Variable name cannot contain `/` character. Variable name is case-insensitive, it is always saved in lower case. When it is specified as mixed case, CMR will convert it into lower case.
 * measurement (REQUIRED): the measurement that the variable belongs to.
-* originator_id (REQUIRED): the Earthdata Login ID of the person who created the variable.
+
+#### <a name="searching-for-variables"></a> Searching for Variables
+
+Variables can be searched for by sending a request to `%CMR-ENDPOINT%/variables`.
+
+Variable search results are paged. See [Paging Details](#paging-details) for more information on how to page through variable search results.
+
+##### Variable Search Parameters
+
+The following parameters are supported when searching for variables.
+
+##### Standard Parameters:
+
+* page_size
+* page_num
+* pretty
+
+##### Variable Matching Parameters
+
+These parameters will match fields within a variable. They are case insensitive by default. They support options specified. They also support searching with multiple values in the style of `variable_name[]=key1&variable_name[]=key2`.
```

Comment:
I updated the doc, will push later. In all the search parameters we support, all the values are ORed unless changed through options. This is the only place that we have it documented. :)